deployments/fargate/README.md

```diff
@@ -5,65 +5,141 @@ for further reading.
 
 Unless stated otherwise, the
 [Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
-(Collector) is deployed to an ECS task in a **sidecar** container. We are using the term
-"sidecar" loosely here to mean having the Collector container as a sidecar to multiple or
-all monitored application containers in the task. We do not mean a Collector sidecar container
-per monitored application container as in the strict sense.
+(Collector) is deployed as a **sidecar** (additional container) to ECS tasks.
 
-Requires Connector release v0.30.0 or newer which corresponds to image tag 0.30.0 and newer.
+Requires Connector release v0.31.0 or newer which corresponds to image tag 0.31.0 and newer.
```

Comment:
This needs to be 32 right? (Requires another release post merge)

Reply:
Yes